src/mono/wasm/runtime/invoke-cs.ts

```diff
@@ -134,6 +248,10 @@ function bind_fn_0V(closure: BindingClosure) {
         try {
             const args = alloc_stack_frame(2);
             // call C# side
+            if (NativeAOT) {
+                (method as any)(args); // TODO MF: handle exception
```

Comment:
```suggestion
                (method as any)(args); // TODO-LLVM-JSInterop: handle exception
```
Unique `TODO` postfixes make it easy to search for them.